### DISCUSSION TITLE
Command input always on top

### DISCUSSION THREAD

**Discussion by khaitranhq** [kovidgoyal/kitty#7209] [Q&A] [answered]
Hi everyone,
I have tried Warp in Linux for a week and I was really interested in the static position of the command prompt but I haven't found the option to do it in kitty yet.
```
> # current command
> # old command
...old command's output
> # older command
...older command's output
....
```

**Comment by kovidgoyal** [comment, marked as answer]
And there never will be such an option. Warp, I assume does that by
running commands in separate ptys and echoing the output. 

The correct place to implement such a feature would be the shell not the
terminal emulator, since the shell is what is responsible for running
commands, not the terminal emulator.
